wradlib/util.py

```diff
@@ -1200,14 +1162,23 @@ def cross_section_ppi(
     method = kwargs.get("method", None)
     tolerance = kwargs.get("tolerance", None)
 
+    # Deprecation warning
+    real_beams = kwargs.get("real_beams")
```

Comment:
Just remove this, if it was removed in 2.0. Better a clean cut now.

Reply:
ok, done